pages/api/recipes/random/index.js

```diff
@@ -0,0 +1,21 @@
+import dbConnect from "../../../../db/connect";
+import Recipe from "../../../../db/models/Recipe";
+
+export default async function handler(request, response) {
+  await dbConnect();
+
+  if (request.method === "GET") {
+    try {
+      const randomRecipes = await Recipe.aggregate([{ $sample: { size: 1 } }]);
+
+      if (!randomRecipes.length) {
```

Comment:
i think you meant either 
`if (!randomRecipes)` or if `(randomRecipes.length === 0)`